sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/output.py

```diff
@@ -117,6 +119,18 @@ def _to_rest_object(self) -> Dict:
         # rest object is generated by extracting Output's properties, see details in to_rest_data_outputs()
         return self._to_dict()
 
+    def _simple_parse(self, value, _type=None):
+        if _type is None:
+            _type = self.type
+        if _type in IOConstants.PARAM_PARSERS:
+            return IOConstants.PARAM_PARSERS[_type](value)
+        return value
+
+    def _normalize_self_properties(self):
+        # parse value from string to its original type. eg: "false" -> False
+        if self.is_control:
```

Comment:
Does "early_available" also need this conversion?

Reply:
Good catch, will add it.